yw_website/yw_website/apps/website/models.py

```diff
@@ -19,7 +19,7 @@ class Tag(models.Model):
         (FILE, "File"),
     )
 
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=32)
```

Comment:
why are we restricting the title lenght?

Reply:
tags should be short